src/main/java/managers/DriverManager.java

```diff
@@ -74,6 +67,18 @@ public WebDriver getDriver() {
         return driver;
     }
 
+    public void waitForPageToLoad() {
+        ExpectedCondition<Boolean> isLoaded = (WebDriver driver) ->
+                ((JavascriptExecutor) driver).executeScript("return document.readyState").toString().equals("complete");
+
+        try {
+            Thread.sleep(1000);
```

Comment:
once again I don't think it should be here, I remember that you used it for a specific project, but I'm against adding it to the framework and slowing down tests for no reason

Reply:
It wasn't for a specific project, it was added because without it the page doesn't even have enough time to start loading, so exception is always thrown. Also, an argument there was that it's not going to slow down tests, because no page will realistically load in less than 1 second
We could delete it of course, but I don't know how - it was the only way I could find to make it work. Do you have some solution for that?